pkg/controlplane/control/port.go

```diff
@@ -35,11 +36,20 @@ const (
 	maxRandomTries = 40
 )
 
+type conflictingTargetPortError struct {
+	port      uint16
+	leaseName types.NamespacedName
+}
+
+func (e conflictingTargetPortError) Error() string {
+	return fmt.Sprintf("port %d is already leased to %v", e.port, e.leaseName)
```

Comment:
```suggestion
	return fmt.Sprintf("port %d is already in use by service %v", e.port, e.leaseName)
```
I think `leased` might be less clear to users